SPECS/python-requests-oauthlib/python-requests-oauthlib.spec

```diff
@@ -9,7 +9,7 @@ Distribution:   Mariner
 
 Name:               python-requests-oauthlib
 Version:            1.2.0
-Release:            7%{?dist}
+Release:            8%{?dist}
```

Comment:
The `%bcond_without python3` can be removed and assumed we always build with Python 3.